tests/unit/components/exhibition/ExhibitionChaptersNavigation.spec.js

```diff
@@ -0,0 +1,76 @@
+import { createLocalVue, mount } from '@vue/test-utils';
+import ExhibitionChaptersNavigation from '../../../../components/exhibition/ExhibitionChaptersNavigation.vue';
+
+const localVue = createLocalVue();
+
+const $route = {
+  params: {
+    chapter: 'astilbe'
+  }
+};
+
+const chapterNavigation = [{
+  name: 'Allium',
+  identifier: 'allium',
+  url: 'en/exhibition/flowers/allium'
+},
+{
+  name: 'Astilbe',
+  identifier: 'astilbe',
+  url: 'en/exhibition/flowers/astilbe'
+},
+{
+  name: 'Calls for education',
+  identifier: 'calls-for-education',
+  url: 'en/exhibition/flowers/calls-for-education'
+}];
+
+const factory = () => mount(ExhibitionChaptersNavigation, {
+  localVue,
+  stubs: ['b-link'],
+  mocks: {
+    $route,
+    localePath: (opts) => opts
+  },
+  propsData: {
+    exhibitionIdentifier: 'flowers',
+    chapterNavigation
+  }
+});
+
+describe('components/exhibition/ExhibitionChaptersNavigation', () => {
+  it('shows a previous and next chapter button', () => {
```

Comment:
Add `context` blocks to explain when each of these expectations should occur, e.g.
```js
context('when there are both previous and next chapters', () => {
  it('has buttons for both', () => {
```